tests/ui/consts/ice-const-size-relate-126359.stderr

```diff
@@ -0,0 +1,18 @@
+error[E0308]: mismatched types
+  --> $DIR/ice-const-size-relate-126359.rs:12:39
+   |
+LL |     let _ = OppOrder::<3, u32> { arr: [0, 0, 0] };
+   |                                       ^^^^^^^^^ expected `3`, found `3`
```

Comment:
This diagnostic still looks buggy.

Reply:
Yeah, it should be suppressed. Will look into it